configure_infrastructure.py

```diff
@@ -0,0 +1,115 @@
+"""
+...
+"""
+import os
+import sys
+from typing import (
+    List,
+)
+
+DOCKER_WAIT = 5
+DOCKERFILE_DATASTORES = [
+    "postgres",
+    "mysql",
+    "mongodb",
+    "mssql",
+]
+EXTERNAL_DATASTORES = [
+    "snowflake",
+    "redshift",
+]
+IMAGE_NAME = "fidesops"
+
+
+def configure_infrastructure(
+    run_application: bool = False,  # Should we run the Fidesops webserver?
+    run_tests: bool = False,  # Should we run the tests after creating the infra?
+    open_shell: bool = False,  # Should we open a bash shell?
+    datastores: List[str] = [],  # Which infra should we create? If empty, we create all
+) -> None:
+    """
+    - Create a Docker Compose file path for all datastores specified in `datastores`.
+    - Defaults to creating infrastructure for all datastores in `DOCKERFILE_DATASTORES` if none
+    are provided.
+    - Optionally runs integration tests against those datastores from the container identified
+    with `IMAGE_NAME`.
+    """
+
+    # Configure docker-compose path
+    path: str = "-f docker-compose.yml"
+    if len(datastores) == 0:
+        os.system(
+            f'echo "no datastores specified, configuring infrastructure for all datastores"'
+        )
+        datastores = DOCKERFILE_DATASTORES + EXTERNAL_DATASTORES
+    else:
+        os.system(f'echo "datastores specified: {", ".join(datastores)}"')
+
+    for datastore in datastores:
+        os.system(f'echo "configuring infrastructure for {datastore}"')
+        if datastore in DOCKERFILE_DATASTORES:
+            # We only need to locate the docker-compose file if the datastore runs in Docker
+            path += f" -f docker-compose.integration-{datastore}.yml"
+        elif datastore not in EXTERNAL_DATASTORES:
+            # If the specified datastore is not known to us
+            os.system(f'echo "Datastore {datastore} is currently not supported"')
+
+    os.system(f'echo "infrastructure path: {path}"')
+    os.system(f"docker-compose {path} build")
+    os.system(f'echo "sleeping for: {DOCKER_WAIT} while infrastructure loads"')
+    os.system(f"sleep {DOCKER_WAIT}")
+
+    # Seed datastores with data
+    for datastore in datastores:
```

Comment:
Nice, this will support datastores other than mssql in future if needed

Reply:
It will yes, so long as we:
- configure the `docker-compose.integration-<env>.yml` file, and
- add `<env>` to the `DOCKERFILE_ENVIRONMENTS` collection atop the file